src/locales/ar/translation.json

```diff
@@ -5,32 +5,26 @@
     "shortformName": "[ar] NYC CTO"
   },
   "location": "[ar] NYC",
+  "navigation": {
+    "dropdowns": [
+      {
+        "label": "[ar] Nav Label 1",
```

Comment:
I think "label" and "simpleLinks" are confusing. Maybe it should be something like "accordionButtonLabel" and "linkLabels"?

Also maybe "parentLinks" should be changed? The label could even just be "links"?

Reply:
For the dropdowns, how about `buttonLabel` and `linkLabels`. And for the parent links, `linkLabels` might be confusing when they're trying to debug error messages and misinterpret which`linkLabels` to fix. How about `parentLinkLabels`, instead?

Reply:
`buttonLabel` and `linkLabels` work for me. I agree that it could be helpful to keep the parent labels different from `linkLabels` but I'm wondering if there's another word besides parent? In our case are they really parent links?

Reply:
`buttonLabel` and `linkLabels` are now used for the dropdowns. Also, I mentioned `parentLinkLabels` since those links are referred to as parent links on the [Storybook](https://trussworks.github.io/react-uswds/?path=/story/components-header--extended-header). What do you think? It's still being referred to as `parentLinks`.

Reply:
I'm not sure what a better term would be, but I am not sure what makes something a parent link. Does it mean linking back to the main page of a website? In the[ landing page template](https://trussworks.github.io/react-uswds/?path=/docs/page-templates-landing-page--landing-page), it just says `simple link` and then `navigation link` in the nav dropdown menus.

Reply:
Also now that `simpleLinks` are just `linkLabels` in the translation files, I don't think we need the word 'simple' in `headerLinks.navDropDownSimpleLinks`. Since the links could already be confusing to someone new to the repo, I think the naming should be even more one-to-one

Reply:
So `headerLinks.navDropDownLinks` instead of `headerLinks.navDropDownSimpleLinks`? And for parent links, let's have `navigation.parentLinkLabels` and `headerLinks.parentLinks`? 

Although, the naming in `src/constants/links.js` seems to be repeating the word `Links` at the end of everything in `headerLinks`.

Reply:
Sounds good. We can update `headerLinks` and `footerLinks` to `header` and `footer` in `src/constants/links.js` then?

Reply:
- [ ] `headerLinks.navDropDownSimpleLinks` → `headerLinks.navDropDownLinks` 
- [ ] `navigation.parentLinks` → `navigation.parentLinkLabels`
- [ ] `headerLinks.parent` → `headerLinks.parentLinks`
- [ ] `headerLinks` --> `header`
- [ ] `footerLinks` --> `footer`